terraform/templates/windows_worker_user_data.tpl

```diff
@@ -42,11 +42,20 @@
   # Add config to HabService.dll.config
   $svcPath = Join-Path $env:SystemDrive "hab\svc\windows-service"
   [xml]$configXml = Get-Content (Join-Path $svcPath HabService.dll.config)
-  $configXml.configuration.appSettings.add[1].value = "${flags}"
-  $configXml.Save((Join-Path $svcPath HabService.dll.config))
 
-  # Enable PIDS_FROM_LAUNCHER feature
-  $env:HAB_FEAT_PIDS_FROM_LAUNCHER=1;
+  # Update the arguments for the Supervisor
+  $launcherArgs = $configxml | select-xml -xpath "//appSettings/add[@key='launcherArgs']"
+  $launcherArgs.Node.value = "${flags}"
+
+  %{ for feature in enabled_features ~}
+
```

Comment:
Remove blank line to be consistent

Reply:
Fixed in https://github.com/habitat-sh/builder/pull/1252/commits/9df4ace531039d2f7894e1e11b3b4f1f5aa3b517